data/functions/itemized.sql

```diff
@@ -16,3 +124,22 @@ begin
     perform retry_processing_schedule_b_records(:START_YEAR_AGGREGATE);
 end
 $$ language plpgsql;
+
+CREATE OR REPLACE FUNCTION rename_table_cascade(table_name TEXT) RETURNS VOID AS $$
+DECLARE
+    child_tmp_table_name TEXT;
+    child_tables_cursor CURSOR (parent TEXT) FOR
+        SELECT c.oid::pg_catalog.regclass AS name
+        FROM pg_catalog.pg_class c, pg_catalog.pg_inherits i
+        WHERE c.oid=i.inhrelid
+            AND i.inhparent = (SELECT oid from pg_catalog.pg_class rc where relname = parent);
+BEGIN
+    EXECUTE format('DROP TABLE IF EXISTS %I CASCADE', table_name);
+    EXECUTE format('ALTER TABLE %1$I_tmp RENAME TO %1$I', table_name);
```

Comment:
In the first statement there is `%I`, which means "treat the interpolated value as a SQL identifier." [[source](https://www.postgresql.org/docs/current/static/functions-string.html#FUNCTIONS-STRING-FORMAT)]

In the second statement, you have `%1$I` and only one argument to the `format` function - what is this interpolating?  It appears to be the same thing, which is why I'm confused that it's not just `%I`. :-)

Based on the lines just below this, it looks like you're just referring to the positional argument explicitly but telling it to still treat it as a SQL identifier?  If so that's fine, I just want to make sure I understand!